File: C++17/백준/Bronze/10813. 공 바꾸기/공 바꾸기.cc
#include <stdio.h>
int main(){
    int N,M,i,j;
    int k;
    int arr[101];

    scanf("%d %d", &N, &M);

    for (k = 1; k <= N; k++){
        arr[k] = k;
    }
    for (k = 1; k <= M; k++){
        scanf("%d %d", &i, &j);
        int temp = 0;
        temp = arr[i];
        arr[i] = arr[j];
        arr[j] = temp;
    }
    for(k = 1; k <= N; k++){
        printf("%d ", arr[k]);
    }

    return 0;
}

File: Java/백준/Bronze/10813. 공 바꾸기/공 바꾸기.java
import java.util.*;
public class Main
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
	    int N, M;
	    N = sc.nextInt();
	    M = sc.nextInt();
	    int[] arr = new int[N];
	    for(int i = 0; i < N; i++){
	        arr[i] = i + 1;
	    }
	    for(int t = 0; t < M; t++){
	        int i = sc.nextInt() - 1;
	        int j = sc.nextInt() - 1;
	        int temp = arr[i];
	        arr[i] = arr[j];
	        arr[j] = temp;
	    }
	    for(int num : arr){
	        System.out.print(num + " ");
	    }
	}
}